numpyro/infer/autoguide.py

```diff
@@ -52,18 +52,18 @@ class AutoGuide(ABC):
 
     :param callable model: a pyro model
     :param str prefix: a prefix that will be prefixed to all param internal sites
-    :param callable init_strategy: A per-site initialization function.
+    :param callable init_loc_fn: A per-site initialization function.
         See :ref:`init_strategy` section for available functions.
     :param callable create_plates: An optional function inputing the same
         ``*args,**kwargs`` as ``model()`` and returning a :class:`numpyro.plate`
         or iterable of plates. Plates not returned will be created
         automatically as usual. This is useful for data subsampling.
     """
 
-    def __init__(self, model, *, prefix="auto", init_strategy=init_to_uniform, create_plates=None):
+    def __init__(self, model, *, prefix="auto", init_loc_fn=init_to_uniform, create_plates=None):
```

Comment:
Since this is a patch release and we have already moved this out of contrib, this will break existing code. We should add a deprecation warning here.

Reply:
Oops, I forgot... :( thanks for reminding, I will add depreciation warning here and Delta too .

Reply:
Actually, could you add `FutureWarning` instead which won't be swallowed by python?